app/code/Magento/Customer/Model/Plugin/CustomerFlushFormKey.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
```

Comment:
Please set copyright same as in other files for `2.1-develop`:

```php
/**
 * Copyright © 2013-2017 Magento, Inc. All rights reserved.
 * See COPYING.txt for license details.
 */
```

Reply:
Changed by:
```
/**
 * Copyright © 2013-2017 Magento, Inc. All rights reserved.
 * See COPYING.txt for license details.
 */
```